File: Java/FizzBuzz.java
import java.util.Scanner;

public class FizzBuzz {
    public static void main(String[] args) {
        for(int dracula = 1; dracula <= 100; dracula++) {
            String output = "";
            if (dracula % 3 == 0)
                output += "Fizz";
            if (dracula % 5 == 0)
                output += "Buzz";
            System.out.println(output.equals("") ? dracula : output);
        }
    }
}

File: Python/FizzBuzz.py
# A likely Camper Solution to Fizzbuzz.  There are more efficient solutions.

for i in range(1, 101, 1):
    if i % 3 == 0 and i % 5 != 0:
        print("Fizz")
    if i % 3 != 0 and i % 5 == 0:
        print("Buzz")
    if i % 3 == 0 and i % 5 == 0:
        print("FizzBuzz")
    else:
        print(i)

